service-discovery/README.adoc

```diff
@@ -31,304 +31,399 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+* A second application that consumes the price updates for publication on a web page
 
+In addition, you have a Kafka instance in {product}. Each time the first application produces a price update, you want to use the Kafka instance to forward the update as an event to the second, consuming application.
 
-In this Quick Start, you'll connect your OpenShift Streams for Apache Kafka cloud service to your OpenShift cluster using the Red Hat OpenShift Application Services. (RHOAS) Operator and CLI. For more information on Red Hat OpenShift Application Services, please visit link:https://cloud.redhat.com/application-services[cloud.redhat.com.]
+To achieve this behavior, you need a way to connect the applications in OpenShift to your Kafka instance in {product}. Connecting a Kubernetes application to a backing service such as {product} is called __service binding__.
 
-.Prerequisites
-ifndef::community[]
-* You have a Red Hat account.
-endif::[]
-* You have a running Kafka instance in {product}.
+This guide describes how to use the Red Hat OpenShift Application Services (RHOAS) Operator to perform service binding for {product}. The Kubernetes platform used throughout this guide is Red Hat OpenShift.
 
-// Condition out QS-only content so that it doesn't appear in docs.
-// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
-ifdef::qs[]
-[#description]
-Using the {product-long} services in your OpenShift cluster{product-long}.
 
-[#introduction]
-Welcome to the {product-long} Service Discovery quick start. In this quick start, you'll learn how to connect your {product} services with https://quarkus.io/[Quarkus^]. Tutorial will connect Kafka with your Application to produce and consume messages from your Kafka instances in {product}.
-endif::[]
+[id="con-about-service-binding-using-rhoas-operator_{context}"]
+== About service binding using the RHOAS Operator
+
+Connecting a Kubernetes application to a backing service such as {product-long} is called __service binding__. To perform service binding for {product}, you use the Red Hat OpenShift Application Services (RHOAS) Operator.
+
+The RHOAS Operator enables you to expose Kafka instances created in {product} to applications in separate OpenShift clusters. You can use commands in the RHOAS CLI that instruct the RHOAS Operator to inject connection credentials for a specified Kafka instance to an application running in OpenShift. With connection between your application and Kafka instance established, you can then work directly with the Kafka instance using standard OpenShift features and APIs.
+
+When you bind a Kafka instance with an OpenShift application, the RHOAS Operator injects connection credentials as files into the Pod for your application. By default, the RHOAS Operator creates the following directory and file structure in the application Pod:
```

Comment:
the bootstrapServer, provider, saslMechanism etc aren't really credentials.

Reply:
credential: "a piece of information that is sent from one computer to another to check that a user is who they claim to be or to allow someone to see information"

Reply:
This incorrect use of credential continues below, but I will leave it to Cmd-F to locate them rather than call them out.

Reply:
Thanks, @pmuir. Yeah, I was looking for a single term to encapsulate 'user', 'password', and everything else. I already had a concern that this might be overloading the definition of _credentials_  :/

Do you think something like_connection parameters_ might be appropriate instead?

Reply:
I think that's much better or "connection details" or something like that

Reply:
Thanks, @pmuir. I had considered _connection details_ or _connection credentials and details_ (too clunky). To me _details_ sound like pieces of static information, rather than parameters that are dynamically used by the application to establish a connection.

I'll go with _connection parameters_, if you're good with that. I'll make the change globally throughout the doc.